### ISSUE TITLE
Custom Jasper Voice?

### ISSUE THREAD

**Issue by BBGDylan** [jasperproject/jasper-client#640]
Hi there . I was wondering, if I recorded someone speaking the entire english vocabulary, would I be able to change jasper's voice into their one?

**Comment by yoginth** [comment]
Yeah it is possible but you need to design your own python library, for synthesis, but it is not suggested better try TTS (Text to Speech) Synthesis.

**Comment by BBGDylan** [comment]
Oh right. TTS is what has preset voices built in, right?

**Comment by G10DRAS** [comment]
some tts permits to create your own voices. and once you created your voice, your machine will be able to sing a song in your voice.
for example below clip is created using tts `festival`
https://github.com/G10DRAS/RoboCop/blob/master/images/baabaablacksheep.wav

**Comment by BBGDylan** [comment]
Huh, cool. What TTS program was used for that clip?
